scripts/utility-wfrp4e.js

```diff
@@ -754,12 +754,13 @@ class WFRP_Utility
 
     if (event.button == 0)
     {
-      if (event.target.text.trim() == game.i18n.localize("ROLL.CritCast"))
+      let mytext = event.target.text || event.target.textContent;
+      if (mytext.trim() == game.i18n.localize("ROLL.CritCast"))
       {
         html = WFRP_Tables.criticalCastMenu($(event.currentTarget).attr("data-table"));
       }
 
-      else if (event.target.text.trim() == game.i18n.localize("ROLL.TotalPower"))
+      else if (mytext.trim() == game.i18n.localize("ROLL.TotalPower"))
```

Comment:
I'm confused, Critical Casting and Total Power have nothing to do with the `/table` menu, but this is supposed to fix it?

Reply:
Yes =, i understand, but when using .text only it is undefined. Hence the .trim() function below failed.
Another solution could be to test if event.target.text is defined, yes.

Reply:
Ah, so throws an exception. Hm I wonder if there's a better way to get the text value, because I've had issues with this in the past, in this same area.